src/99-appendices/05-units.md

```diff
@@ -1,39 +1,76 @@
-| second         | s           | time                                       |
-| ampere         | A           | electric current                           |
-| kelvin         | K           | thermodynamic temperature                  |
-| mole           | mol         | amount of substance                        |
-| candela        | cd          | luminous intensity                         |
-| radian         | rad         | angle                                      |
-| steradian      | sr          | solid angle                                |
-| hertz          | Hz          | frequency                                  |
-| newton         | N           | force, weight                              |
-| pascal         | Pa          | pressure, stress                           |
-| joule          | J           | energy, work, heat                         |
-| watt           | W           | power, radiant flux                        |
-| coulomb        | C           | electric charge or quantity of electricity |
-| volt           | V           | voltage (electrical potential), emf        |
-| farad          | F           | capacitance                                |
-| ohm            | Ω           | resistance, impedance, reactance           |
-| siemens        | S           | electrical conductance                     |
-| weber          | Wb          | magnetic flux                              |
-| tesla          | T           | magnetic flux density                      |
-| henry          | H           | inductance                                 |
-| degree Celsius | °C          | temperature relative to 273.15 K           |
-| lumen          | lm          | luminous flux                              |
-| lux            | lx          | illuminance                                |
-| becquerel      | Bq          | radioactivity (decays per unit time)       |
-| gray           | Gy          | absorbed dose (of ionizing radiation)      |
-| sievert        | Sv          | equivalent dose (of ionizing radiation)    |
-| katal          | kat         | catalytic activity                         |
+The [CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12) convention
+for encoding units is REQUIRED to achieve maximum portability and limited
+variability of representation.
+In case that a CMIXF representation of SI units is not possible, you will have
+to declare your units as custom units and define them in accompanying JSON
+files, as described in the [units section](../02-common-principles.md#units).
+Earlier versions of the BIDS standard listed the following Unicode symbols, and
+these are still included for backwards compatibility:
+
+1.  `U+00B5` (µ)
+1.  `U+00B0` (°)
+1.  `U+2126` (Ω).
+
+It is REQUIRED that units be CMIXF-12 compliant or among these three Unicode
+characters.
+Please take care to not confuse the accepted unicode `U+00B5` (µ) with the
+unicode NOT accepted `U+03BC` (μ):
```

Comment:
Minor grammar issues, and added links to a useful reference that can give additional context.

```suggestion
Please take care not to confuse the accepted Unicode character
[`U+00B5`](https://codepoints.net/U+00B5) (µ) with the disallowed Unicode
character [`U+03BC`](https://codepoints.net/U+03BC) (μ):
```